src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -54,6 +72,32 @@ private static CloudBillingClient cloudBillingClient() {
         }
     }
 
+    private static CloudBillingClient cloudBillingClient() {
+        return cloudBillingClient(null);
+    }
+
+    public static boolean verifyAccess(AuthenticatedUserRequest user, String billingAccountId) {
+        ResourceName resource = BillingAccountName.of(billingAccountId);
+        List<String> permissions = Collections.singletonList("billing.resourceAssociations.create");
```

Comment:
It would be great to have a constant for magic strings like this. It's too bad Google hasn't provided them